unified-codes/apps/data-service/src/app/v1/types/DgraphDataSource.ts

```diff
@@ -175,6 +176,12 @@ export class DgraphDataSource extends RESTDataSource {
     }
   }
 
+  private static getPropertyType(property: IProperty): string {
+    const { type: types } = property;
+    const [type] = types ?? [];
+    return type;
+  }
```

Comment:
`IProperty` is incorrect - with the change to use `dgraph.type` the `type` is now an array, which is why this change was needed.

Reply:
Added d02119336 to rename `dgraph.type` to `types` and map that to `type` in `IProperty`
It's not great, but it is clearer.

Reply:
Now I get it! I think it's better the way you had it before. A comment would be plenty good enough, I think (rather than adding another `types` field, which is arguably a bit more confusing!). Sorry for the extra work 😢 !

Reply:
Now who's messing with whom??! this switch over to use `dgraph.type` and therefore an array of types was your move originally!

Reply:
Revert and comment in 174db11fe

Reply:
> 
> 
> Now who's messing with whom??! this switch over to use `dgraph.type` and therefore an array of types was your move originally!

https://youtu.be/UmzsWxPLIOo.